crates/oxc_prettier/src/lib.rs

```diff
@@ -47,4 +48,9 @@ impl<'a> Prettier<'a> {
         let doc = program.format(&mut self);
         Printer::new(doc, self.source_text, self.options).build()
     }
+
+    pub fn cmds(mut self, program: &Program<'a>) -> Vec<Command<'a>> {
```

Comment:
I think we just want the `Doc`, commands are not interesting :-/

Reply:
https://github.com/oxc-project/oxc/pull/1350/commits/44bf9a293f5e202a539b7d25621d26681eb42de7